ajax/libs/jQuery-linkify/package.json

```diff
@@ -13,16 +13,17 @@
     "linkify"
   ],
   "homepage": "https://github.com/SoapBox/linkifyjs/",
```

Comment:
[This website](http://soapbox.github.io/linkifyjs/) might be a better choice to be used as the homepage info.